src/main/java/games/strategy/triplea/ui/BattlePanel.java

```diff
@@ -442,11 +442,11 @@ public void bombingResults(final GUID battleId, final List<Die> dice, final int
 
   class CenterBattleAction extends AbstractAction {
     private static final long serialVersionUID = -5071133874755970334L;
-    Territory m_territory;
+    Territory battleSite;
```

Comment:
I renamed this field to something totally different to avoid a potential shadowing issue with `CenterBattleAction$MyTimerTask#territory` below.